src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/LoadingFields.cs

```diff
@@ -140,5 +140,18 @@ static void Main()
             CompileAndVerify(compilation, expectedOutput: @"Value1
 Value2");
         }
+
+        [Fact]
+        public void TestLoadFieldsOfReadOnlySpanFromCorlib()
+        {
+            var comp = CreateCompilation("", targetFramework: TargetFramework.Net60);
+
+            var readOnlySpanType = comp.GetTypeByMetadataName("System.ReadOnlySpan`1");
+            Assert.NotNull(readOnlySpanType);
```

Comment:
>Assert.NotNull(readOnlySpanType); [](http://example.com/codeflow?start=12&length=33)

Does internal ```GetTypeByMetadataName``` API ever return null?

Reply:
According to nullability annotations, yes. But considering your suggestion below, this is no longer relevant